sp800-63-3/sec1_2_introduction.md

```diff
@@ -70,13 +70,13 @@ Transactions* [[ESIG]](#ESIG) for additional information on legal risks, especia
 
 Additionally, Federal agencies implementing these guidelines should
 adhere to the requirements of Title III of the E-Government Act,
-entitled the [Federal Information Security Modernization Act](#FISMA), and related NIST standards and guidelines. FISMA directs Federal agencies to develop, document, and implement
+entitled the *Federal Information Security Modernization Act* [FISMA](#FISMA), and related NIST standards and guidelines. FISMA directs Federal agencies to develop, document, and implement
```

Comment:
need double brackets around FISMA to make it appear in single brackets (since the outer brackets make it a link and disappear)